### DISCUSSION TITLE
oflg/Fishing

### DISCUSSION THREAD

**Discussion by giscus** [tiddly-gittly/TiddlyWiki-CPL#14] [Plugins]
# oflg/Fishing

https://tw-cpl.netlify.app/#oflg%2FFishing:oflg%2FFishing%20Index

**Comment by oflg** [comment]
我是插件的开发者，有任何使用问题和建议，欢迎评论和交流。

I am the developer of the plugin, if you have any questions and suggestions on the use of the plugin, please feel free to comment and exchange.

**Reply by jefferyp141k** [reply]
您好，感谢您的贡献。我安装时遇到了困难。 
Incremental Learning program based on TiddlyWiki
$:/plugins/oflg/fishing
2.4.6
 安装后无法使用。敬请回复。

**Reply by linonetwo** [reply]
看到你在qq群里问的了，就是 tw 核心版本需要升级的那个对吧
